.github/workflows/e2e.yaml

```diff
@@ -51,6 +51,12 @@ jobs:
         uses: fluxcd/pkg/actions/kubectl@main
         with:
           version: 1.21.2
+      - name: Setup SOPS
```

Comment:
Please create an actions in pkg for sops based on this https://github.com/fluxcd/pkg/blob/main/actions/kustomize/action.yml